DiscountDiscover/View Controllers/MapViewController.m

```diff
@@ -24,10 +25,18 @@ - (void)viewDidLoad {
     CLLocationCoordinate2D locationCoordinate = [LocationManager sharedLocationManager].currentLocationCoordinate;
 
     GMSCameraPosition *camera = [GMSCameraPosition cameraWithLatitude:locationCoordinate.latitude longitude:locationCoordinate.longitude zoom:17];
-    GMSMapView *mapView = [GMSMapView mapWithFrame:self.view.frame camera:camera];
-    mapView.settings.myLocationButton = YES;
-    mapView.myLocationEnabled = YES;
-    [self.view addSubview:mapView];
+    self.mapView = [GMSMapView mapWithFrame:self.view.frame camera:camera];
+    self.mapView.settings.myLocationButton = YES;
+    self.mapView.myLocationEnabled = YES;
+    [self.view addSubview:self.mapView];
+    
+    [self addDealMarkers];
```

Comment:
If you only add the markers in `viewDidLoad` they won't get updated according to the list. Seems like you might want to move this to viewWillAppear or something later so the map always shows the same as the list. 
Also, are you planning to fetch new deals when user scrolls or zooms the map? In this case, would you also update the list view accordingly or should they be independent?

Reply:
Ah I realized and fixed this in a commit right after this PR unfortunately! And I wasn't planning on it, but that would definitely be a good thing to look into next week.